src/chainlifter/lifter.py

```diff
@@ -10,14 +10,14 @@
 import chainlifter._core as _core
 
 
-class Strand(StrEnum):
+class Strand(str, Enum):
```

Comment:
Oh we should revert this in our other repos where we don't have CI testing different python versions.